actually:

~~~
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.14.1:testCompile (default-testCompile) on project oak-auth-ldap: Compilation failure: Compilation failure:
[ERROR] /C:/projects/apache/oak/trunk/oak-auth-ldap/src/test/java/org/apache/jackrabbit/oak/security/authentication/ldap/impl/LdapIdentityProviderTest.java:[28,20] cannot find symbol
[ERROR]   symbol:   class Matchers
[ERROR]   location: package org.hamcrest
[ERROR] /C:/projects/apache/oak/trunk/oak-auth-ldap/src/test/java/org/apache/jackrabbit/oak/security/authentication/ldap/impl/LdapIdentityProviderTest.java:[113,17] cannot find symbol
[ERROR]   symbol:   variable Matchers
[ERROR]   location: class org.apache.jackrabbit.oak.security.authentication.ldap.impl.LdapIdentityProviderTest
[ERROR] /C:/projects/apache/oak/trunk/oak-auth-ldap/src/test/java/org/apache/jackrabbit/oak/security/authentication/ldap/impl/LdapIdentityProviderTest.java:[114,25] cannot find symbol
[ERROR]   symbol:   variable Matchers
[ERROR]   location: class org.apache.jackrabbit.oak.security.authentication.ldap.impl.LdapIdentityProviderTest
[ERROR] /C:/projects/apache/oak/trunk/oak-auth-ldap/src/test/java/org/apache/jackrabbit/oak/security/authentication/ldap/impl/LdapIdentityProviderTest.java:[115,25] cannot find symbol
[ERROR]   symbol:   variable Matchers
[ERROR]   location: class org.apache.jackrabbit.oak.security.authentication.ldap.impl.LdapIdentityProviderTest
[ERROR] /C:/projects/apache/oak/trunk/oak-auth-ldap/src/test/java/org/apache/jackrabbit/oak/security/authentication/ldap/impl/LdapIdentityProviderTest.java:[116,32] cannot find symbol
[ERROR]   symbol:   variable Matchers
[ERROR]   location: class org.apache.jackrabbit.oak.security.authentication.ldap.impl.LdapIdentityProviderTest
[ERROR] /C:/projects/apache/oak/trunk/oak-auth-ldap/src/test/java/org/apache/jackrabbit/oak/security/authentication/ldap/impl/LdapIdentityProviderTest.java:[117,32] cannot find symbol
[ERROR]   symbol:   variable Matchers
[ERROR]   location: class org.apache.jackrabbit.oak.security.authentication.ldap.impl.LdapIdentityProviderTest
[ERROR] /C:/projects/apache/oak/trunk/oak-auth-ldap/src/test/java/org/apache/jackrabbit/oak/security/authentication/ldap/impl/LdapIdentityProviderTest.java:[118,32] cannot find symbol
[ERROR]   symbol:   variable Matchers
[ERROR]   location: class org.apache.jackrabbit.oak.security.authentication.ldap.impl.LdapIdentityProviderTest
[ERROR] /C:/projects/apache/oak/trunk/oak-auth-ldap/src/test/java/org/apache/jackrabbit/oak/security/authentication/ldap/impl/LdapIdentityProviderTest.java:[119,32] cannot find symbol
[ERROR]   symbol:   variable Matchers
[ERROR]   location: class org.apache.jackrabbit.oak.security.authentication.ldap.impl.LdapIdentityProviderTest
[ERROR] /C:/projects/apache/oak/trunk/oak-auth-ldap/src/test/java/org/apache/jackrabbit/oak/security/authentication/ldap/impl/LdapIdentityProviderTest.java:[121,45] cannot find symbol
[ERROR]   symbol:   variable Matchers
[ERROR]   location: class org.apache.jackrabbit.oak.security.authentication.ldap.impl.LdapIdentityProviderTest
[ERROR] /C:/projects/apache/oak/trunk/oak-auth-ldap/src/test/java/org/apache/jackrabbit/oak/security/authentication/ldap/impl/LdapIdentityProviderTest.java:[121,32] cannot find symbol
[ERROR]   symbol:   variable Matchers
[ERROR]   location: class org.apache.jackrabbit.oak.security.authentication.ldap.impl.LdapIdentityProviderTest
~~~

--- a/oak-auth-ldap/pom.xml
+++ b/oak-auth-ldap/pom.xml
@@ -166,6 +166,12 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-library</artifactId>
+            <version>1.3</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>jul-to-slf4j</artifactId>
             <scope>test</scope>

